templates/cemanager.yaml

```diff
@@ -37,9 +37,9 @@ spec:
       # terminationGracePeriodSeconds should be set to the upper bound for container rebuilds and creates.
       # 5 minutes
       terminationGracePeriodSeconds: 300
-      # Start the manager with a specific service account
-      # that only has permissions to what it needs in order to
-      # administer pods.
+      # Start the manager with a specific service account with no permissions.
+      # The envproxy forwards a service account token to the cemanager on
+      # startup called "coder" with all the permissions it needs.
```

Comment:
So envproxy is now the sole owner of the service account (that does anything), as in that service account will be the separate envproxy helm chart?

Reply:
This will be the same in the envproxy chart. Envproxy will assume the `coder` SA, and forward that to the cemanager.